packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -588,7 +588,10 @@ class ListTile extends StatelessWidget {
       return selectedColor ?? tileTheme.selectedColor ?? theme.listTileTheme.selectedColor ?? theme.colorScheme.primary;
     }
 
-    final Color? color = iconColor ?? tileTheme.iconColor ?? theme.listTileTheme.iconColor;
+    final Color? color = iconColor
+      ?? tileTheme.iconColor
+      ?? theme.listTileTheme.iconColor
+      ?? (theme.useMaterial3 ? theme.colorScheme.onSurface.withOpacity(0.38) : null);
```

Comment:
Can we add a comment describing where the `onSurface.withOpacity` came from in the spec as this is not going through the usual token template system.

Reply:
Thanks! I added a comment for the color, similar to the `icon_button.dart` docs.
https://github.com/flutter/flutter/blob/d8fdb83555522dbf25659e6509ccc3551bcc0193/packages/flutter/lib/src/material/icon_button.dart#L799

Reply:
The disabled case is handled above on line 589. So this seems to be returning the incorrect color for enabled, non-selected list tile icons.

Reply:
`line 589` disabled color is for disabled `list_tile`, this wasn't the fix.

The wording was wrong in the issue and PR, apolgies.

By disabled, it means unselected. This is the author's code, `ListTile` is not disabled.
```dart
     home: Scaffold(
        body: Column(
          children: const [
            ListTile(
              leading: Icon(Icons.add_rounded),
            ),
            Icon(Icons.add_rounded),
          ]
        ),
      ),
```

> The disabled case is handled above on line 589. So this seems to be returning the incorrect color for enabled, non-selected list tile icons.

In this case, I agree. At the time there were no specs for the list item but we have now and I can see the difference you're talking about.

I did a quick comparison here with Flutter vs Compose.

| Flutter | Compose |
| --------------- | --------------- |
| <img src="https://user-images.githubusercontent.com/48603081/197767366-f96bbc30-ce93-46f3-bc6a-adda0ead5832.jpg"  /> | <img src="https://user-images.githubusercontent.com/48603081/197767361-04a6cca8-c9f6-4551-8708-292188bd9946.jpg" />


`ListTile` needs a complete migration, just for unselected color.


Filed https://github.com/flutter/flutter/issues/114006, please take a look.

Reply:
Thanks, I think we'll be able to workaround this internally.